refrigerator.cs

```diff
@@ -110,23 +106,20 @@ public void deletExpiredItems(List<Item> allItem)
         {
             List<Item> expiredItems = new List<Item>();
```

Comment:
also here, remove the definition of the list to item obj and use it as property.